nikola/conf.py.in

```diff
@@ -243,6 +243,12 @@ POSTS_SECTIONS = True
 # are the default and will apply GENERATE_ATOM if set.
 # POSTS_SECTIONS_ARE_INDEXES = True
 
+# Final locations are:
+# output / TRANSLATION[lang] / SECTION_PATH / SECTION_NAME / index.html (list of posts for a section)
+# output / TRANSLATION[lang] / SECTION_PATH / rss.xml (RSS feed for a section)
+# (translatable)
```

Comment:
`SECTION_PATH` is *not* translatable in your implementation. You need to add it to `nikola/nikola.py`, both to the default config and to `TRANSLATABLE_SETTINGS`. Then, when you use it, you have to provide the language (see comment below). (Search for `TAG_PATH` in `nikola.py` to see how it is done.)